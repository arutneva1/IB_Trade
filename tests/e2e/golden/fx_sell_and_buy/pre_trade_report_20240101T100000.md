NetLiq: 3240.74
Cash CAD: 1000.00
Cash USD: 740.74
Cash Buffer: 7.41

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 50.00 | 0.00 | 5000.00 | 100.00 | 1620.37 | 16.2037 | BUY | 1620.37 |  |
| BBB | 50.00 | 77.32 | -2731.96 | 50.00 | -885.36 | -17.7072 | SELL | -885.36 |  |
| TOTAL | 100.00 | 77.32 | 2268.04 |  | 735.01 |  |  | 735.01 |  |
