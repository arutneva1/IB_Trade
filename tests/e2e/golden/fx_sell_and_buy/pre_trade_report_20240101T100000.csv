NetLiq,3240.74
Cash CAD,1000.00
Cash USD,740.74
Cash Buffer,7.41

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional,reason
AAA,50.0,0.0,5000.0,100.0,1620.37,16.2037,BUY,1620.37,
BBB,50.0,77.32,-2731.96,50.0,-885.36,-17.7072,SELL,-885.36,
TOTAL,100.0,77.32,2268.04,,735.01,,,735.01,
